### PR TITLE
[RES-733] Fix reported issues for Sendgrid google java class - JediBees

### PR DESCRIPTION
**Pull request by danielamihalache**

### CODE REVIEW

File: Sendgrid.java [hunk 1]
```diff
@@ -120,6 +121,7 @@ public Sendgrid addToName(String name) {
         return this;
     }
 
+
```
Comment (efimovalex) [inline Sendgrid.java]: delete new line

File: Sendgrid.java [hunk 2]
```diff
@@ -142,6 +144,27 @@ public Sendgrid setFrom(String email) {
     }
 
     /**
+     * getFrom - Get the from email address
```
Comment (efimovalex) [inline Sendgrid.java]: update comments to match the described function

File: Sendgrid.java [hunk 3]
```diff
@@ -142,6 +144,27 @@ public Sendgrid setFrom(String email) {
...
+     *
+     * @return  The from email address
+     */
+    public String getFromName() {
+        return this.from_name;
+    }
+
+    /**
+     * setFrom - Set the from email
```
Comment (efimovalex) [inline Sendgrid.java]: update comments to match the described function

File: Sendgrid.java (hunk 1 above)
Comment (danielamihalache) [inline Sendgrid.java]: done

File: Sendgrid.java (hunk 2 above)
Comment (danielamihalache) [inline Sendgrid.java]: done

File: Sendgrid.java (hunk 3 above)
Comment (danielamihalache) [inline Sendgrid.java]: done

**Pull request merged**